### PR TITLE
Modified mocks and type CompareResultsItem

### PR DESCRIPTION
**Pull request by beatrice-acasandrei**
Frontend part of https://mozilla-hub.atlassian.net/browse/PCF-272.
Contains:
- [x] added base_rev and new_rev fields to CompareResultsItem
- [x] removed function that capitalizes the value of confidence_text, the backend will send the  capitalized value
- [x] deleted unnecessary fields: confidence_text_long (these values are kept in `/src/resources/Strings.tsx`), t_value_confidence, t_value_care_min
- [x] updated the `processResults` function so that it can differentiate between results based on the new revision

### CODE REVIEW

File: src/reducers/CompareResults.ts
Comment (julienw) [inline src/reducers/CompareResults.ts:5]: Not for this patch: we should probably use async imports so that the bundler put these away from the main bundle.

File: src/components/CompareResults/beta/ResultsTable.tsx
Comment (julienw) [inline src/components/CompareResults/beta/ResultsTable.tsx:28]: 20y in this industry and I just learnt about `String.prototype.concat` :-)

**Review by julienw — APPROVED** [review]
`added base_rev and new_rev fields to CompareResultsItem`
Can you give some more information about that? Is it new from the server or was it already present in the server response before? If it's new, is it already deployed?

Otherwise this looks good to me :)

**Comment by beatrice-acasandrei** [comment]
> `added base_rev and new_rev fields to CompareResultsItem` Can you give some more information about that? Is it new from the server or was it already present in the server response before? If it's new, is it already deployed?
> 
> Otherwise this looks good to me :)

You can look at https://mozilla-hub.atlassian.net/browse/PCF-272. I've added the changes to the response concomitantly with this PR, which has already been merged, but not yet deployed. I will ask tomorrow for a deploy so that Carla can have access to the new response when she's back on Monday.

File: src/reducers/CompareResults.ts
Comment (beatrice-acasandrei) [reply src/reducers/CompareResults.ts:5]: I'll add a ticket for that, thanks.

**Comment by julienw** [comment]
OK then it could be better to wait until it's deployed to merge this PR, otherwise https://beta--mozilla-perfcompare.netlify.app and local development will be broken.

**Pull request merged**
